core/src/main/java/org/apache/carbondata/core/scan/result/vector/CarbonColumnVector.java

```diff
@@ -114,4 +114,14 @@
 
   void setLazyPage(LazyPageLoader lazyPage);
 
+  CarbonColumnVector getColumnVector();
+
+  CarbonColumnVector getChildrenVector();
```

Comment:
Make this interface a **list** of CarbonColumnVector considering other complex type

Reply:
done

Reply:
done